Day-I-Social-Card/social-card/src/components/HeaderComponents/HeaderContainer.js

```diff
@@ -1,2 +1,17 @@
 import React from 'react';
 import './Header.css';
+import ImageThumbnail from "./ImageThumbnail";
+import HeaderTitle from "./HeaderTitle";
+import HeaderContent from "./HeaderContent"
+const HeaderContainer = () => {
+    return (
+        <div className = "header-container">
+        <ImageThumbnail />
+        <div className = 'header-content' >
+            <HeaderTitle />
```

Comment:
You can put the `HeaderTitle` in the `HeaderContent` and just add `HeaderContent` to the header container instead. One less thing to move with CSS.